src/AppBundle/Entity/Product.php

```diff
@@ -63,9 +63,15 @@ class Product
      */
     private $images;
 
+    /**
+     * @ORM\OneToMany(targetEntity="OrderProduct", mappedBy="product", cascade={"remove", "persist"})
```

Comment:
pas de remove en cascade pour les produits !
il faudra ajouter un état is deleted pour les supprimer.